requirements.txt

```diff
@@ -1,41 +1,43 @@
-pip==9.0.3
+pip==19.0.3
 Django==1.11.20 # rq.filter: <1.12
--e git+https://github.com/OpenDataServices/flatten-tool.git@v0.7.0#egg=flattentool
--e git+https://github.com/open-contracting/lib-cove-ocds.git@v0.2.1#egg=libcoveocds
--e git+https://github.com/OpenDataServices/lib-cove-web.git@v0.6.0#egg=libcoveweb
--e git+https://github.com/OpenDataServices/lib-cove.git@v0.3.1#egg=libcove
-django-bootstrap3==9.1.0
-django-debug-toolbar==1.9.1
+-e git+https://github.com/OpenDataServices/flatten-tool.git@524f437abe48d7f9b8a2d6a1cf6fa9a49d002b0f#egg=flattentool
+-e git+https://github.com/open-contracting/lib-cove-ocds.git@8a3d696eb614063bdec1790c0e3fda68d0445040#egg=libcoveocds
+-e git+https://github.com/OpenDataServices/lib-cove-web.git@9e75c72f7a7a255aad10d100360e85fae35f2d94#egg=libcoveweb
+-e git+https://github.com/OpenDataServices/lib-cove.git@52acb2f0dcd296b9d952ff2adbae61dc4f8dd296#egg=libcove
```

Comment:
Is it not better to have the versions instead of the commits?

Reply:
Maybe, but this is due to the behaviour of Pip and is not something that would be easy to change. When you `pip freeze', you get the commit hash.

Reply:
It can be changed manually afterwards.

Reply:
It's an extra step that I don't think gives a lot of benefits - we can just check in req.in if we want to know.

Let's discuss this under a general discussion around "best practices" in a Loomio thread or something - we should agree one way between us then all stick to it consistently